Imgui/CMakeLists.txt

```diff
@@ -2,7 +2,12 @@ cmake_minimum_required (VERSION 3.6)
 
 project(Diligent-Imgui CXX)
 
-set(DEAR_IMGUI_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../ThirdParty/imgui")
+if (NOT DILIGENT_TOOLS_DEAR_IMGUI_PATH)
```

Comment:
I'd suggest little more concise `DILIGENT_DEAR_IMGUI_PATH`

Reply:
That's a valid point. I've changed it.